eng/common/scripts/Package-Properties.ps1

```diff
@@ -15,12 +15,18 @@ class PackageProps
     [boolean]$IsNewSdk
     [string]$ArtifactName
     [string]$ReleaseStatus
+    [string[]]$Namespaces
```

Comment:
You don't need to add a new property here you can add a member dynamically to the object.